### ISSUE TITLE
Add "Ratio" to list of key words in measure name to be automatically formatted to percentage.

### ISSUE THREAD

**Issue by ruve1k** [DaxStudio/DaxStudio#341]
> [If the column is a decimal number AND the name includes “pct” or “%” then use the format “0.00%”](https://darren.gosbell.com/2020/06/dax-studio-how-measure-formatting-works/?utm_source=feedly&utm_medium=rss&utm_campaign=dax-studio-how-measure-formatting-works)

Request: Automatically Format Results to percentage format when measure name contains "**ratio**".

Also, consider default percentage format to only use one decimal place instead of two. (I.e. 0.0% instead of 0.0%)

**Comment by dgosbell** [comment]
I think "ratio" is probably a debatable one. I've seen ratios express both as a percentage and as a raw decimal. And the number of decimals to display is probably personal preference also. So if you don't mind I might edit your request from just making changes to the existing hard coding to extend this feature so that the user can edit the list of "percentage words" in the options screen and also choose other options like specifying the number of decimal places.

**Comment by ruve1k** [comment]
Yes, I agree with your recommendation of editing the request to enable editing the list of "percentage words.
Also, an option to manually input the desired format strings would be great.
